servant/servant.cabal

```diff
@@ -98,7 +98,7 @@ library
   -- We depend (heavily) on the API of these packages:
   -- i.e. re-export, or allow using without direct dependency
   build-depends:
-      http-api-data          >= 0.4      && < 0.4.1
+      http-api-data          >= 0.4      && < 0.4.2
```

Comment:
We depend on tight range, as the API is re-exported.

Please bump lower bounds to `>=0.4.1`

Reply:
Updated!